packages/cactus-plugin-ledger-connector-fabric-socketio/package.json

```diff
@@ -30,6 +34,8 @@
     "socket.io": "4.4.1"
   },
   "devDependencies": {
+    "@hyperledger/cactus-api-client": "1.0.0",
+    "@hyperledger/cactus-cmd-api-server": "1.0.0",
```

Comment:
@outSH The API server does not explicitly depend on the plugins, but it is used to install them at runtime so I would recommend not creating a (hidden) circular dependency this way. 
When I made the self signed PKI generator I was lazy and didn't put it in a separate package but I should've so if you want to duplicate it in one of the test packages (test-tooling is my recommendation) then we can create an issue to remove the old one from the API server package in the future (breaking change, v2.0.0, etc.) and until then it can be just marked as deprecated.

Reply:
socketio-based validators has nothing to do with `cactus-cmd-api-server` plugin system, but I agree that it's good to keep up a convention, and that this logic belongs somewhere else. I've moved the PKI generation to test-tooling, and marked the old implementation as deprecated. Please review the last commit, I'll squash it if everything is OK.

Reply:
@outSH Sounds good, thank you for adjusting and very sorry for the slow review. I'm trying to bring the CI back online.